ironfish/src/rpc/clients/client.ts

```diff
@@ -733,6 +734,24 @@ export abstract class RpcClient {
       ).waitForEnd()
     },
 
+    getParsedConsensusParameters: (
```

Comment:
I'm not sure this is the best approach

1. This is duplicating code of the `getConsensusParameters`
2. `number | null` is a better type, especially for consumers of the SDK (modifying ConsensusParameters itself is a topic for another day..)
3. This is a new pattern that deviates from the expectations of the existing pattern - RPC client functions return RPC responses - leading to an inconsistent API for consumers of the SDK

If we do need this, my initial feeling is that this should be a util function

Reply:
Ok yea I can just duplicate it in the 2 places we call it. I was back and forth on this